core/controllers/admin.py

```diff
@@ -631,3 +632,21 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        if feconf.CAN_SEND_EMAILS:
+            response = {
+                'msg': 'Success! Mail sent to admin.',
+            }
+            email_manager.send_dummy_mail_to_admin()
```

Comment:
Can you also include the username of the sender in the email? I think that would be useful for verifying that the admin gets only the expected emails.

Reply:
Done.